fv3core/stencils/del2cubed.py

```diff
@@ -103,6 +107,26 @@ def __init__(self, grid):
         self._compute_meridional_flux = StencilWrapper(func=compute_meridional_flux)
         self._update_q = StencilWrapper(func=update_q)
 
+        self._copy_full_domain = StencilWrapper(
+            func=copy_defn, origin=origin, domain=domain
+        )
+        self._copy_corners_x = StencilWrapper(
+            func=corners.copy_corners_x_stencil_in_out,
+            origin=(self.grid.isd, self.grid.jsd, 0),
+            domain=(self.grid.nid, self.grid.njd, self.grid.npz),
+            externals={
+                **ax_offsets,
+            },
+        )
+        self._copy_corners_y = StencilWrapper(
+            func=corners.copy_corners_y_stencil_in_out,
+            origin=(self.grid.isd, self.grid.jsd, 0),
+            domain=(self.grid.nid, self.grid.njd, self.grid.npz),
+            externals={
```

Comment:
I know you are just using what was there already, but these can be self,.grid,full_domain() and self.grid.domain_shape_full()

Reply:
this should change in the oopified version.